Online-vs-Offline-IDE.md

```diff
@@ -0,0 +1,69 @@
+**\#Online Offline Editors/IDE**
+
+ 
+
+\*\*There are several offline IDE that programmers use in order to stay more
+productive to learn and be more productive at work\*\*
+
+ 
+
+1.Eclipse is a free editor that is both online and offline incorporates Java,
+C/C++, JavaScript, and PHP by IBM. Its \*biggest advantages are code completion
+by tabbing method which saves a lot of time writing documentations. It \*has a
+build in syntax check to fix any mistype words while coding a project. All of
+features we've come to expect \*of an IDE are their code completion, templates,
```

Comment:
Reading long, information dense paragraphs are difficult. Reader loses attention trying to break the wall of text.

As a thumb rule, don't write more than 2 sentences per para. 3 at most.